tools/points-service/readme.md

```diff
@@ -0,0 +1,68 @@
+
+# MEV-Commit Points Service
+
+This repository implements an Ethereum event listener and points accrual service for validators across different registries (Vanilla, Middleware/Symbiotic, and AVS/EigenLayer). It also exposes a simple HTTP API (see [`api.go`](./api.go)) that provides an external points interface—primarily for Symbiotic usage ([details here](https://symbioticfi.notion.site/External-Points-API-Specification-16981c079c178059b6b7e9740cf987a8)).
+
+---
+
+## Vanilla Registry Events
+
+The Vanilla Registry emits a `Staked` event when a validator is registered through direct ETH staking:
+
+```solidity
+event Staked(
+    address indexed sender,     // The account that called stake()
+    address withdrawalAddress,  // The address that can withdraw/unstake 
+    bytes pubkey,              // The validator's BLS public key
+    uint256 amount             // Amount of ETH staked
+);
+```
+
```

Comment:
What about `Unstaked(msg.sender, validator.withdrawalAddress, pubKey, validator.balance)` and/or `StakeWithdrawn(msg.sender, expectedWithdrawalAddress, pubKey, balance)`?